### DISCUSSION TITLE
TLS/SSL Server support

### DISCUSSION THREAD

**Discussion by Cube-Controls** [khoih-prog/EthernetWebServer_SSL#13] [General]
Just wondering did you have any success in adding TLS/SSL Server support ?

**Comment by khoih-prog** [comment]
Thanks for the interest in the library. But I have no intention to add TLS/SSL Server support.

The main reason is the regular MCUs are **not powerful enough** to handle the TLS/SSL server encoding/decoding job (for multiple clients, pages, etc).

That's the job for PC or small MPU, such as RPi Pi, etc.

Hopefully in the future, there in on the market the Ethernet chip / board handling the TLS/SSL encoding/decoding job inside the chip / board.

Now, try the WiFi, such as ESP32, WiFiNINA, to see if there is any TLS/SSL server support.

**Reply by Cube-Controls** [reply]
Thanks for respond.
